description: do not use pointers after calling realloc (gcc 12 -Wuse-after-free)
author: Michael Gilbert <mgilbert@debian.org>

--- a/dlls/winex11.drv/clipboard.c
+++ b/dlls/winex11.drv/clipboard.c
@@ -1853,6 +1853,7 @@ static BOOL X11DRV_CLIPBOARD_GetProperty
 
     for (;;)
     {
+        unsigned char *tmp = val;
         if (XGetWindowProperty(display, w, prop, pos, INT_MAX / 4, False,
                                AnyPropertyType, atype, &aformat, &nitems, &remain, &buffer))
         {
@@ -1862,7 +1863,7 @@ static BOOL X11DRV_CLIPBOARD_GetProperty
         }
 
         count = get_property_size( aformat, nitems );
-        *data = realloc( val, pos * sizeof(int) + count + 1 );
+        *data = realloc( tmp, pos * sizeof(int) + count + 1 );
 
         if (!*data)
         {
